keypad/keypad.c

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright (c) 2020 Raspberry Pi (Trading) Ltd.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include <stdio.h>
+#include "pico/stdlib.h"
+
+
+#define KEYPAD_NUM_ROWS           4
+#define KEYPAD_NUM_COLUMNS        4
+#define NO_KEY_PRESSED         '\0'  // Just a character that unlikely be
+                                     // present in keyboards, so it can by used
+                                     // to determine if no key is pressed.
+#define READ_MS_DELAY            10  // Delay between lectures to avoid noise
```

Comment:
"lectures" ??

Reply:
Oops! My brain does not completely switch from Spanish to English sometimes

Fixed as part of https://github.com/raspberrypi/pico-examples/pull/542/commits/52a7f7b842428f11266c69d9ea44aedddfbe5d70